index.html

```diff
@@ -3743,55 +3753,50 @@ <h2>DID Resolution</h2>
 be a conformant <a>DID</a> as defined in <a href="#did-syntax"></a>.
             </dd>
             <dt>
-did-resolution-input-metadata
+options
             </dt>
             <dd>
-A <a href="#metadata-structure">metadata structure</a> consisting of input
-options to the <code>resolve</code> and <code>resolveRepresentation</code>
-functions in addition to the <code>did</code> itself. Properties defined by this
-specification are in <a href="#did-resolution-input-metadata-properties"></a>.
-This input is REQUIRED, but the structure MAY be empty.
+A <a href="#metadata-structure">metadata structure</a> containing properties
+defined in <a href="#did-resolution-options"></a>. This input is
+REQUIRED, but the structure MAY be empty.
             </dd>
         </dl>
 
         <p>
-The output variables of these functions are as follows:
+The return value of <code>resolve()</code> is <dfn>Resolution</dfn> which is
+a <a data-cite="INFRA#maps">map</a> that contains the
+<a>didResolutionMetadata</a>, <a>didDocument</a>, and
+<a>didDocumentMetadata</a> properties. The return value of
+<code>resolveRepresentation()</code> is <dfn>ResolutionRepresentation</dfn>
+which is a <a data-cite="INFRA#maps">map</a> that contains
+<a>didResolutionMetadata</a>, <a>didDocumentStream</a>, and
+<a>didDocumentMetadata</a> properties. These properties are described below:
         </p>
 
         <dl>
             <dt>
-did-resolution-metadata
+<dfn>didResolutionMetadata</dfn>
             </dt>
             <dd>
-              <p>
 A <a href="#metadata-structure">metadata structure</a> consisting of values
-relating to the results of the <a>DID resolution</a> process. This structure
-is REQUIRED and MUST NOT be empty.
-              </p>
-              <p>
-This metadata typically changes between invocations of the
-<code>resolve</code> and <code>resolveRepresentation</code> functions as it
-represents data about the resolution process itself. Properties defined by
```

Comment:
Is there a reason for removing this? This is meant to describe the difference between didResolutionMetadata and didDocumentMetadata.